script.js

```diff
@@ -44,45 +59,59 @@ function displayPacman() {
 displayWorld()
 displayPacman()
 
+// I might call this CURRENT_ROW
 let currX = 13
+// I might call this CURRENT_COLUMN
 let currY = 5
 let coins = 0;
 
+// cuz hard to remember for humanz
+let LEFT_ARROW = 37
+let UP_ARROW = 38
+let RIGHT_ARROW = 39
+let DOWN_ARROW = 40
+
+// also hard to remember for humanz
+let DOWN_ROTATION = 'rotate(180deg)' 
+let UP_ROTATION = 'rotate(270deg)'
+let LEFT_ROTATION = 'rotate(90deg)'
+let RIGHT_ROTATION = 'rotate(360deg)'
+

```

Comment:
Another little bit that can help keep things from becoming overwhelming:

```javascript
let LEFT_ARROW = 37
let UP_ARROW = 38
let RIGHT_ARROW = 39
let DOWN_ARROW = 40

// Its nice to keep functions small, a good rule of thumb is 
// "only do one thing".  if the onKeyDown function's job is just
// "figure out what key was called" then it asks another function 
// to do other work.
// Not a hard rule or anything, but 5-10 lines for a function is 
// often a nice guideline, but break it when necessary.
document.onkeydown = function(e) {
  if(e.keyCode == LEFT_ARROW) {
    movePacManLeft()
  } else if (e.keyCode == RIGHT_ARROW) {
    movePacManRight()
  } else if (e.keyCode == UP_ARROW) {
    movePacManUp()
  } else if (e.keyCode == DOWN_ARROW) {
    movePacManDown()
  }
}
```

Reply:
I noticed you repeat this chunk 4 times also:

```javascript
      if(!isWall){
        pacman.x += 44
        currX += 1;
      }
```
but it is the same code each time, so you could pull it out, and do it only once, at the end of the set of `if else if else...`.